bundles/pubsub/test/CMakeLists.txt

```diff
@@ -173,12 +173,6 @@ if (BUILD_PUBSUB_PSA_UDP_MC)
     add_test(NAME pstm_deadlock_udpmc_test COMMAND pstm_deadlock_udpmc_test WORKING_DIRECTORY $<TARGET_PROPERTY:pstm_deadlock_udpmc_test,CONTAINER_LOC>)
     setup_target_for_coverage(pstm_deadlock_udpmc_test SCAN_DIR ..)
 
-    #TCP Endpoint test is disabled because the test is not stable when running on Travis
```

Comment:
Was this removed intentionally?

Reply:
Yes. If you look closely, it's trying to add a TCP test inside of the UDP block. The TCP block also adds this test, it must've somehow got duplicated.